governance-and-automation/automated-workflows/set-up-automated-workflows.md

```diff
@@ -39,179 +39,42 @@ When dealing with automated workflows, you can either:
-* **Minimum Number of Owners** - when enabled and applied to Microsoft Teams and Microsoft 365 Groups, **this policy triggers an automated workflow when the team/group has fewer owners than defined in the policy settings**
-* **Maximum Number of Owners** - when enabled and applied to Microsoft Teams and Microsoft 365 Groups, **this policy triggers an automated workflow when the team/group has more owners than defined in the policy settings**
-* **Orphaned Resources** - when enabled and applied to Microsoft Teams and Microsoft 365 Groups, **this policy triggers an automated workflow when the team/group has no active owners**
+## Policies
 
-In this article, you can **find information on how to create, enable, and apply** all mentioned policy types, along with **explanations for all policy options.**
+SysKit Point enables you to create and apply three types of ownership policies that make sure your **Microsoft Teams and Microsoft 365 Groups comply with your company's policy regarding ownership** as well as a tenant-wide policy that **requires Guest User Access validation**. 
 
-For more information on how **to resolve specific policy violations**, **navigate to the appropriate article given below**:
-* [Resolve Minimum Number of Owners policy violation](minimum-number-of-owners.md)
-* [Resolve Maximum Number of Owners policy violation](maximum-number-of-owners.md)
-* [Resolve Orphaned Resources policy violation](orphaned-resources.md)
+Take a look at the articles below to learn how to set up each governance policy:
 
-## Guest Users Expiration Policy
-With SysKit Point, you can **enable a tenant-wide policy** that will **require Guest User Access validation periodically or when SysKit Point detects that a guest user is inactive**. 
+* [**Minimum Number of Owners**](minimum-number-of-owners-admin.md) - when enabled and applied to Microsoft Teams and Microsoft 365 Groups, this policy **triggers an automated workflow when the team/group has fewer owners** than defined in the policy settings
+* [**Maximum Number of Owners**](maximum-number-of-owners-admin.md) - when enabled and applied to Microsoft Teams and Microsoft 365 Groups, this policy **triggers an automated workflow when the team/group has more owners** than defined in the policy settings
+* [**Orphaned Resources**](orphaned-resources-admin.md) - when enabled and applied to Microsoft Teams and Microsoft 365 Groups, this policy **triggers an automated workflow when the team/group has no active owners**
+* [**Guest Users Expiration**](guest-users-expiration-admin.md) - periodically checks Guest Users' access, or when SysKit Point detects a guest user is inactive
 
-In this article, you can **find information on how to enable the policy**, along with detailed **descriptions of all policy options.**
+By clicking on the names of each policy, you can **find an article with information on creating and enabling** all mentioned policy types. 
+
+How to apply the created policies is described below. 
```

Comment:
u ovom članku konkretno nije nigdje opisano kako applyati policy na timove, siteove i grupe
taj dio bi bilo dobro još dodati

Reply:
u ovom članku bi trebali linkati i na access requestove što je također policy. za sad možeš samo linkati na postojeći governance-and-automation\access-requests\create-access-requests-policy.md članak, ali moguće da ćemo u kasnijoj fazi cijeli access-requests folder staviti pod automated workflows, ali za sad samo linkaj na taj postojeći članak

Reply:
ista stvar kao i za access requests policy vrijedi i za access review policy. samo linkati na governance-and-automation\permissions-review\create-apply-access-review-policies.md za sad